blackduck/HubRestApi.py

```diff
@@ -1197,6 +1197,36 @@ def remove_version_as_component(self, main_project_release, sub_project_release)
     # Code locations or Scans Stuff
     #
     ###
+    
+    def download_project_scans(self, project_name,version_name, output_folder=None):
+        version = self.get_project_version_by_name(project_name,version_name)
+        codelocations = self.get_version_codelocations(version)
+        import os
+        if output_folder:
+            if not os.path.exists(output_folder):
+                os.makedirs(output_folder, 0o755, True)
+        
+        result = []
+        
+        for item in codelocations['items']:
+            links = item['_meta']['links']
+            matches = [x for x in links if x['rel'] == 'enclosure']
+            for m in matches:
+                url = m['href']
+                filename = url.split('/')[6]
+                if output_folder:
+                    pathname = os.path.join(output_folder, filename)
+                else:
+                    pathname = filename
+                import requests
```

Comment:
redundant?

Reply:
there is not import requests at the head of the file, so I put it inline, close with the meat and away from the boilerplate code.